index.js

```diff
@@ -9,17 +9,34 @@ const tmpDir = require('./tmp-dir');
 
 const inputFileName = core.getInput('file-name');
 const awsOutputDir = core.getInput('aws-output-directory');
-const outputFileName = inputFileName.replace(/\.xd$/, `.${awsOutputDir}`);
+const conversionType = core.getInput('conversion-type');
+let outputFileName = inputFileName.replace(/\.xd$/, `.${awsOutputDir}`);
+let filePath = path.join(tmpDir.name, inputFileName);
+if (conversionType === 'F2S') {
+  outputFileName = `${inputFileName}.sketch`;
```

Comment:
Cant you use `awsOutputDir` here as in L13?

Reply:
Done !